nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -14,6 +14,20 @@ These release notes track the development of the {run-once-operator} for {produc
 
 For an overview of the {run-once-operator}, see xref:../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {run-once-operator}].
 
+[id="run-once-duration-override-operator-release-notes-1-1-2"]
+== Run Once Duration Override Operator 1.1.2
+
+Issued: 31 October 2024
+
+The following advisory is available for the {run-once-operator} 1.1.2:
+
+* link:https://access.redhat.com/errata/RHSA-2024:8337[RHSA-2024:8337]
```

Comment:
This looks good on this line (and that's how most places do it), but it looks like the old 1.1.1 one doesn't do it like this. I'd fix the 1.1.1 one to follow this format.